Source/WebKit/Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.mm

```diff
@@ -27,7 +27,7 @@
 
 #import <WebKit/WKMain.h>
```

Comment:
Maybe add the `objc` include headers here instead of in the `AuxiliaryProcessExtensionBridge.h` header.

Reply:
I removed the objc header in the latest patch, since it was not required.